testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/web/ssl/HTTPSWebConnectorTestCase.java

```diff
@@ -320,8 +320,8 @@ public void stopContainer() throws Exception {
         deployer.undeploy(APP_CONTEXT);
         final ModelControllerClient client = TestSuiteEnvironment.getModelControllerClient();
         final ManagementClient managementClient = new ManagementClient(client, TestSuiteEnvironment.getServerAddress(),
-                TestSuiteEnvironment.getServerPort(), "http-remoting");
-        LOGGER.trace("*** reseting test configuration");
+                TestSuiteEnvironment.getServerPort(), "remote+http");
+        LOGGER.info("*** reseting test configuration");
```

Comment:
Was there a reason to change this to info instead of trace?

Reply:
I don't remember except that the test was failing at some point

Reply:
@ehsavoie I think make it trace again